scene/register_scene_types.cpp

```diff
@@ -93,6 +93,7 @@
 #include "scene/gui/color_rect.h"
 #include "scene/gui/control.h"
 #include "scene/gui/dialogs.h"
+#include "scene/gui/exposed_container.h"
```

Comment:
While this is a draft it isn't really possible to evaluate or consider when depending on files that aren't included

Reply:
Apologies. That's leftover from a test, will clean that up.  Unless you think I should remove this pr draft until it's in a better state. Don't want to waste people's time and all that.

Reply:
No problems! Do remove this code so this can be evaluated and assisted with